tests/test_stream_no_parsing.py

```diff
@@ -0,0 +1,514 @@
+"""

```

Comment:
these test cases need revising and simplifying. You need to check the actual output, not just validate that the number of lines is correct (_I appreciate you probably picked this up from one of my own lazier test cases! - see comment above_). You'll need to update the EXPECTED_RESULTS entries to reflect the raw binary output, but you don't need to go overboard - just one test case per protocol (NMEA, UBX. RTCM), or if you prefer a single test case combining all 3 protocols. That will certainly make the DIFF a lot smaller!

Reply:
I did update the no parsing test for both the iterator and the read method for a mixed messages (RTCM, UBX and NMEA)